content/docs/tools-ui/branding-guidelines.md

```diff
@@ -0,0 +1,53 @@
+---
+linkTitle: Branding Guidelines
+title: Open 3D Engine's Branding Guidelines
+description: A guide for using Open 3D Engine's (O3DE) brand logo, brand color, and other resources in the BlueJay Design System. 
+weight: 10
+---
+
+![The page banner for O3DE Branding Guideline](/images/tools-ui/branding-guidelines/page-banner.png)
+
+Welcome to the brand guidelines page for **Open 3D Engine (O3DE)**. Here you will find the O3DE style guide, asset downloads, and any other resources we use in the design system for O3DE.
+
+If you feel like something is incorrect or missing here, you can [file an issue](https://github.com/o3de/sig-ui-ux/issues/new/choose) with the UI/UX Special Interest Group on GitHub. Feel like talking design? We welcome everyone to our open chat room on [Discord](https://discord.com/invite/xNb2q4SJKJ) at #SIG-UX-UI. Our community is always open to chat anything related to user experience and user interface!
+
+## Available resources
+
+- [O3DE Branding Guidelines PDF](#o3de-branding-guidelines)
+- [O3DE Logo Pack](#o3de-logo-pack)
+- [BlueJay Design System Logo Pack](#bluejay-design-system-logo-pack)
+
+## O3DE Branding Guidelines
+
+This is a PDF for O3DE's brand guidelines. It contains details on our primary and secondary logos, various other brand assets, and instructions for correct applications of the O3DE Logo. It also has a breakdown of our color palette if you are interested in developing marketing materials that showcase a product using O3DE.
+
+[{{< image-width "/images/tools-ui/branding-guidelines/brand-guideline.png" "700" "The dos and don'ts of using the O3DE logo." >}}](/files/Open-3D-Branding-Guidelines-June-2021.pdf)  
+
+<br>
+
+[Download O3DE Branding Guidelines PDF](/files/Open-3D-Branding-Guidelines-June-2021.pdf)
+
+## O3DE Logo Pack
+
+The O3DE Logo Pack contains all the essential files that a designer or developer needs for the O3DE logo in one convenient zip file. These brand assets all come with standard JPEGs, transparent backgrounds, and editable vectors for designers.
+
+The pack file also also already includes the above O3DE brand guidelines.
```

Comment:
Suggest being more crisp on the contents of the zip and eliminating the word "editable". Users shouldn't be editing the assets.

```suggestion
The O3DE Logo Pack contains all the essential brand assets for the O3DE logo in one convenient zip file. These brand assets are available as JPEGs,  PNGs with transparent backgrounds, and SVG and EPS vector files.

The O3DE Logo Pack also also includes the above O3DE brand guidelines PDF.
```